### PR TITLE
python312Packages.openllm: 0.5.7 -> 0.6.10; python312Packages.bentoml: 1.2.18 -> 1.3.3

### PR DESCRIPTION
**Pull request by natsukium**
## Description of changes

Diff: https://github.com/bentoml/OpenLLM/compare/refs/tags/v0.5.7...v0.6.10
Changelog: https://github.com/bentoml/OpenLLM/releases/tag/v0.6.10

Diff: https://github.com/bentoml/BentoML/compare/refs/tags/v1.2.18...v1.3.3
Changelog: https://github.com/bentoml/BentoML/releases/tag/v1.3.3

also, remove `openllm-core` and `openllm-client` since the project's philosophy has changed.
https://github.com/bentoml/OpenLLM/releases/tag/v0.6.0

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 336924` at 9655d459 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages marked as broken and skipped:</summary>
  <ul>
    <li>openllm</li>
    <li>python311Packages.bentoml</li>
    <li>python312Packages.bentoml</li>
  </ul>
</details>

**Comment by natsukium** [comment]
`openllm` will be fixed by https://github.com/NixOS/nixpkgs/pull/336903

File: pkgs/by-name/op/openllm/package.nix [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    typer
+    uv
+  ];
+
+  # no tests
+  doCheck = false;
+
+  pythonImportsCheck = [ "openllm" ];
+
+  meta = with lib; {
```
Comment (drupol) [inline pkgs/by-name/op/openllm/package.nix:65]: Shouldn't we add `meta.mainProgram` set to `openllm` ?

Comment (natsukium) [reply pkgs/by-name/op/openllm/package.nix:65]: good catch, thanks.

**Review by drupol — APPROVED** [review]

**Pull request merged**